tests/0080-admin_ut.c

```diff
@@ -1833,11 +1895,39 @@ static void do_test_ListConsumerGroupOffsets(const char *what,
 
         rd_kafka_event_destroy(rkev);
 
+
+        TEST_SAY(
+            "Call ListConsumerGroupOffsets with topic-partition list"
+            "containing duplicates.\n");
+        rd_kafka_ListConsumerGroupOffsets(rk, cgoffsets_duplicate, 1, options,
+                                          q);
+        rd_kafka_ListConsumerGroupOffsets_destroy_array(cgoffsets_duplicate,
+                                                        MY_LIST_CGRPOFFS_CNT);
+        /* Poll result queue */
+        rkev = rd_kafka_queue_poll(q, exp_timeout + 1000);
+        TEST_SAY("ListConsumerGroupOffsets: got %s\n",
+                 rd_kafka_event_name(rkev));
+
+        /* Expecting error */
+        err = rd_kafka_event_error(rkev);
+        TEST_ASSERT(err, "expected ListConsumerGroupOffsets to fail");
+
+        errstr_ptr = rd_kafka_event_error_string(rkev);
+        TEST_ASSERT(!strcmp(errstr_ptr, "Duplicate partitions not allowed"),
```

Comment:
Shall we use `rd_strcmp`?

Reply:
If you see in tests it's not used, it adds a check for null but in this case cannot be null because of the previous assert

Reply:
Sure for here.

General comment - I think we should always think about using rd_* version of the function as we might have to implement things differently based on the architecture or version.